doc/man3/EVP_PKEY_encapsulate.pod

```diff
@@ -41,7 +41,10 @@ unless I<genkeylen> is NULL.
 If I<wrappedkey> is not NULL and the call is successful then the
 internally generated key is written to I<genkey> and its size is written to
 I<*genkeylen>. The encapsulated version of the generated key is written to
-I<wrappedkey> and its size is written to I<*wrappedkeylen>.
+I<wrappedkey> and its size is written to I<*wrappedkeylen>.  Note that if
+I<unwrappedlen> is not NULL, then the value it points to must hold the size of
```

Comment:
unwrappedlen??

Reply:
Not sure what you mean here.  The api call definition is:
```
int EVP_PKEY_decapsulate(EVP_PKEY_CTX *ctx,
                          unsigned char *unwrapped, size_t *unwrappedlen,
                          const unsigned char *wrapped, size_t wrappedlen);
```
and unwrappedlen is the 3rd parameter.

Reply:
@nhorman This is EVP_PKEY_encapsulate documentation.

Reply:
fixed